Resources/assets/css/layout/sidebar.css

```diff
@@ -0,0 +1,39 @@
+/*
+ * sidebar menu styles ?
+ * -------------------- */
+
+.container-fluid > .sidebar {
+    top: auto;
+}
+
+.sidebar-search {
+    padding: 15px;
+}
+
+.arrow {
+    float: right;
+}
+
+.fa.arrow:before {
+    content: "\f104";
+}
+
+.active > a > .fa.arrow:before {
+    content: "\f107";
+}
+
+/*
+ * Not used anywhere ?
+ * -------------------- */
+.nav-second-level li,
+.nav-third-level li {
+    border-bottom: none !important;
+}
+
+.nav-second-level li a {
+    padding-left: 37px;
+}
+
+.nav-third-level li a {
+    padding-left: 52px;
+}
```

Comment:
Looks like the corresponding code was removed in f5383def0323f1796125bc6580ec1836127083c1